cmd/kubeapps-apis/server/server.go

```diff
@@ -34,16 +35,28 @@ import (
 	pluginsGRPCv1alpha1 "github.com/kubeapps/kubeapps/cmd/kubeapps-apis/gen/core/plugins/v1alpha1"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/reflection"
+	"google.golang.org/grpc/status"
 	"google.golang.org/protobuf/encoding/protojson"
 	log "k8s.io/klog/v2"
 )
 
+// LogRequest is a gRPC UnaryServerInterceptor that will log the API call
+func LogRequest(ctx context.Context, req interface{}, info *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (response interface{}, err error) {
+	start := time.Now()
+	res, err := handler(ctx, req)
+	log.Infof("LogRequest - %s %v %s\n",
+		time.Since(start),
+		status.Code(err),
```

Comment:
Just wondering: are we sure `err` will always exist? I mean, what if it is `nil` ?

Also: should we move it to another file, or we better do that once we have more interceptors configured?

Reply:
`status.Code(nil)` is OK: https://play.golang.org/p/6JbcUA1jYAX